### DISCUSSION TITLE
i cant see email sender via emailjs

### DISCUSSION THREAD

**Discussion by maghradz3** [emailjs-com/help#92] [Q&A]
I integrated emailjs in my portfolio app, employer send me email from my portfolio app via emailjs. I get this email on my mail box, but there is not shown sender email. Please help me to solve this problem and get email of sender

**Comment by xr0master** [comment]
You can use dynamic variables to ask for the email and add it to the email content.

**Reply by EmadArif** [reply]
How ?
